### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by canary255**
Transition added

### CODE REVIEW

File: Game/Rotation.pde [hunk 1]
```diff
@@ -1,34 +1,65 @@
...
+
+/* ACTUALMENTE SUSTITUIDO
+ private float getRotationFromMouse() {
+ return map(mouseX, 0, width, -PI, PI);
+ 
+ }*/
+
+
+
+private float getRotation() {
```
Comment (azawadzka) [inline Game/Rotation.pde]: change name to getRotationFromJoystick

File: Game/Rotation.pde [hunk 2]
```diff
@@ -1,34 +1,65 @@
...
-        mp = 1;
-        mr = 0;
-    } else {
-        mp = 0;
-        mr = 1;
-    }
-    return new int[] {mp, mr};
-}
+public float ROTATION; // [-PI, PI]
+int angulo;
```
Comment (azawadzka) [inline Game/Rotation.pde]: Is this the angle of the torch? please give it a more expressive name like VERTICAL_ANGLE (and in English). I'm not sure though if it's an angle at all, maybe just TORCH_HEIGHT?

File: Game/Game.pde [hunk 3]
```diff
@@ -52,27 +77,92 @@ void next_level() {
...
+
+  /*if ((key == 'B' || key == 'b') && room.player_can_unlock()) { //room.getLockedObject() != null &&  && room.getLockedObject().isUnlockable()
+    board.free_element(room.item_p, room.item_r);    
+    player.inventory.removeItem(room.getItem());
+    player.inventory.display();
+  }*/
+}
 
+
+void transicion() {
```
Comment (azawadzka) [inline Game/Game.pde]: please English names

**Review by Rednaxelus — CHANGES_REQUESTED** [review]
What @azawadzka said.

Also the cam must be changed to be optional by @RubenGQ97 first before merging to master

**Pull request merged**